packages/virtual/src/index.js

```diff
@@ -13,7 +14,49 @@ export default function virtual(modules) {
   return {
     name: 'virtual',
 
+    options(options) {
+      let updatedInput = options.input;
```

Comment:
@lukastaegert please give this a look. I recall something recently about this hook not being ideal but I'm having a hard time recalling what it was.